Emerald.CoreX/Emerald.CoreX.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
And even then, it's not particularly any different from the existing Core .net framework project - what's the point?

Reply:
I can agree on that. As I said, after we re-create the CoreX again, we can totally delete the existing `Emerald.Core` and rename `Emerald.CoreX` to  `Emerald.Core`. What do you think?

Reply:
LGTM!